src/stars.css

```
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:nth-last-child(n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-last-child(n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-last-child(n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-last-child(n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-last-child(n + 1) {
+  background-image: url(./images/star-active.svg);
+}
```

Use comma instead of duplicate same
```suggestion
.stars--1 .stars__star:nth-last-child(n + 5),
.stars--2 .stars__star:nth-last-child(n + 4),
.stars--3 .stars__star:nth-last-child(n + 3),
.stars--4 .stars__star:nth-last-child(n + 2),
.stars--5 .stars__star:nth-last-child(n + 1) {
  background-image: url(./images/star-active.svg);
}
```

---

thanks, especially for "Use commas instead of equals". This recording option was not obvious to me.